src/pages/MyResources.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { FC, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { RouteProps } from 'react-router-dom';
+import { getMyResources } from '../api/resourceApi';
+import { CircularProgress, List, ListItem, ListItemIcon, ListItemText, Typography } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { Resource } from '../types/resource.types';
+import ImageIcon from '@material-ui/core/SvgIcon/SvgIcon';
+
+const StyledPageContent = styled.div`
+  display: grid;
+  justify-items: center;
+`;
+
+const MyResources: FC<RouteProps> = (props) => {
+  const { t } = useTranslation();
+  const [isLoadingMyResources, setIsLoadingMyResources] = useState(true);
+  const [resourcesUnpublished, setMyUnpublishedResources] = useState<Resource[]>([]);
+  const [resourcesPublished, setMyPublishedResources] = useState<Resource[]>([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const promises: Promise<any>[] = [];
+      promises[0] = getMyResources().then((response) => {
```

Comment:
trenger ikke pakke inn i promise-array. Det ble gjort på ressurs-visning pga mange kall.

Reply:
Sant, du har helt rett. Tenkte å endre på det, men det gjør det kanskje lettere å legge til andre kall senere om det får være slik? (dlr-gui-backend-resources-sharings/v1/sharings/users/authorized skal også kalles for å hente "resources shared with me" - dersom vi skal bry oss om det)

Reply:
Vi kommer til å endre alle api-kall til async/await snart - så du kan gjerne droppe Promisen :-)

Reply:
Oki :)